src/tmx/TmxUtils/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ ENDIF ()
 
 TARGET_INCLUDE_DIRECTORIES (${PROJECT_NAME} SYSTEM PUBLIC
                             $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>
+                            $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src/telemetry>
```

Comment:
Why is this line necessary for the new telemetry sub-package but not for the existing kafka, simulation, database sub-pacakges

Reply:
I think that is not the case. I think https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/455ed7d9e4d178649e75dde1db4b59cea474ac70/src/tmx/TmxUtils/CMakeLists.txt#L2
```cmake
FILE (GLOB_RECURSE HEADERS "src/" "*.h*")
```
recursively goes through all the subdirectories so I don't think this is necessary at all. Looking at the documentation here https://cmake.org/cmake/help/v3.7/command/file.html for GLOB_RECURSE it seems that it "will traverse all the subdirectories of the matched directory and match the files."

Reply:
When you remove this line what error do you get?

Reply:
Cannot compile because cannot find that header file